### PR TITLE
Add access date to Natural Earth citations

### LINKED ISSUE #743: Indicate date of access for NE layers
https://github.com/nsidc/qgreenland/issues/738#issuecomment-1683079513

### PR DESCRIPTION
**Pull request by MattF-NSIDC**
## Description

Closes #743 

## Checklist

If an item on this list is done _or not needed_, check it with `[x]` or click the
checkbox.

- [x] The PR description links to issues that it resolves with `closes #{issue_number}`
- [x] Config lockfile updated (`inv config.export > qgreenland/config/cfg-lock.json`)
- [x] Environment lockfile updated if needed (`conda-lock`)
- [x] Version bumped if needed (`bumpversion (major|minor|patch|prerelease|build)`)
- [x] CHANGELOG.md updated (for user-facing changes)
- [x] Documentation updated if needed
- [x] New unit tests if needed

### CODE REVIEW

**Comment by MattF-NSIDC** [comment]
Citation for NE layers look like:

```
Citation:
Made with Natural Earth [Accessed on: 2023-08-18]
```

File: qgreenland/config/datasets/geus_mineral_occurrences.py [hunk 1]
```diff
@@ -30,8 +30,8 @@
                 """Thorning, L., Christensen, L. A., Dawes, P. R., Garde, A. A.,
                 Heijboer, T. C., Kalvig, P., Larsen, L. M., Larsen, U., Nielsen, T. F.,
                 Rehnström, E. F., Thomassen, B., Thrane, K., Schjøth, F. & Secher, K. "Greenland
-                mineral occurrences map (GMOM)", GEUS GeoNetwork catalogue. Accessed on
```
Comment (trey-stafford) [inline qgreenland/config/datasets/geus_mineral_occurrences.py:33]: Is there a reason for removing the "Accessed on" bit? This makes it a little less ambiguous. E.g., the GrIMP suggested citation has "Date Accessed" before the date.

File: qgreenland/config/datasets/soil_types.py [hunk 2]
```diff
@@ -39,7 +39,7 @@
                 Arnoldussen, O. Arnalds, and M. Yli-Halla. 2002. Northern
                 Circumpolar Soils Map, Version 1.  Greenland. Ottawa, Canada.
                 Research Branch, Agriculture and Agri-Food Canada. doi:
-                https://doi.org/. {{date_accessed}}."""
+                https://doi.org/. {{date_accessed}}"""
```
Comment (trey-stafford) [inline qgreenland/config/datasets/soil_types.py:42]: What's the doi here :thinking:

File: tasks/config.py [hunk 3]
```diff
@@ -37,7 +37,7 @@ def validate(ctx, verbose=False):
     print("🎉🦆 Configuration validation passed.")
 
 
-@task
+@task(aliases=["lock"])
```
Comment (trey-stafford) [inline tasks/config.py:40]: :rocket:

File: qgreenland/config/datasets/soil_types.py (hunk 2 above)
Comment (trey-stafford) [reply qgreenland/config/datasets/soil_types.py:42]:
```suggestion
                https://doi.org/10.7265/eb1s-4551. {{date_accessed}}"""
```

**Review by trey-stafford — APPROVED** [review]
Great! One comment about text around the access date that we may want to consider first, but otherwise looks good.

File: qgreenland/config/datasets/geus_mineral_occurrences.py (hunk 1 above)
Comment (MattF-NSIDC) [reply qgreenland/config/datasets/geus_mineral_occurrences.py:33]: @trey-stafford check out the change to the `util/metadata.py` module. My reasoning is that the text around access date was very inconsistent. Sometimes there was no text to tell readers what that date was. Now, `{{date_accessed}}` no longer returns only a date string, but the whole shebang, e.g.: `[Accessed on: 2023-08-18]`.

Comment (trey-stafford) [reply qgreenland/config/datasets/geus_mineral_occurrences.py:33]: Ah, I missed that! Thanks you :sweat_smile:

**Review by trey-stafford — APPROVED** [review]

**Pull request merged**